templates/etc/ssh/sshd_config.j2

```diff
@@ -156,10 +160,6 @@ AllowGroups {{ (sshd_allow_groups|d([]) + sshd_group_allow_groups|d([]) + sshd_h
 
 {% endif %}
 
-{% if sshd_custom_options|d() %}
-# Custom options
-{{ sshd_custom_options }}
-
 {% endif %}
```

Comment:
Sorry about that. I know why you have an empty line before the endif but it can lead to such problems.